README.md

```diff
@@ -40,6 +40,29 @@ Ensure packages are installed with correct version numbers by running:
   npm install --save react-dates moment@>=#.## react@>=#.## react-dom@>=#.## react-addons-shallow-compare@>=#.##
   ```
 
+#### Include component
+```
+import { SingleDatePicker } from 'react-dates';
+```
+
+#### Include CSS
+When you use Webpack with CSS loader:
+```
+import 'react-dates/lib/css/_datepicker.css';
+```
+Otherwise create a CSS file with the contents of `node_modules/react-dates/lib/css/_datepicker.css` and include it in the html `<head>` section.
+
+#### Make some awesome datepickers
+```
+<SingleDatePicker
+  id="date_input"
+  date={this.state.date}
+  focused={this.state.focused}
+  onDateChange={(date) => this.setState({ date })}
```

Comment:
@ljharb means that these should be `(date) => { this.setState({ date }); }` and `({ focused }) => { this.setState({ focused }); }` :)